pkg/monitor/resourcewatch/storage/git_store.go

```diff
@@ -90,12 +91,19 @@ func NewGitStorage(path string) (*GitStorage, error) {
 	if err != nil {
 		return nil, err
 	}
-	storage := &GitStorage{path: path, repo: repo}
+	storage := &GitStorage{path: path, repo: repo, fileMode: os.ModePerm}
```

Comment:
this is 0777 and above is 0644, why differnt?

Reply:
For posterity, the original value is / was 0777.  The code change kept the original but added an opt in change to 0644.  Since we are the only ones using it we don't really need to preserve the original.  It just seemed like a nice thing to do in case others wanted the other / different value at some point.